tools/hpo/configs/process.yaml

```diff
@@ -0,0 +1,19 @@
+# Process config example for dataset
```

Comment:
The format of HPO config or how to write a HPO config should be introduced in HPO README files.